reproducible/binaries.sha256sum

```diff
@@ -0,0 +1,9 @@
+c182bb4902fff51b2f56810fc2a27df3646cd66ba21359162354d53445623ab8	third_party/tock/target/thumbv7em-none-eabi/release/nrf52840dk.bin
```

Comment:
I don't think this file is meant to be checked in the repository.

Reply:
@gendx Should we keep this file in the repository?

Reply:
That was unintended, like the binary. I'll remove them ASAP.